src/libraries/System.Private.CoreLib/src/System/Reflection/MethodBase.cs

```diff
@@ -207,6 +207,11 @@ BindingFlags invokeAttr
                         }
                         else
                         {
+                            if (arg != null && sigType.IsNullableOfT && sigType.GetGenericArguments()[0].IsEnum)
```

Comment:
We should add a comment explaining that we need this workaround due to lack of support for `Nullable<Enum>` default values.